server/src/session.ts

```diff
@@ -239,6 +240,7 @@ export class Session {
           triggerCharacters: ['<', '.', '*', '[', '(', '$', '|']
         },
         definitionProvider: true,
+        typeDefinitionProvider: true,
```

Comment:
This means we will have to also implement `getTypeDefinitionAtPosition` for View Engine.
Please make the change in `@angular/language-service` first before merging this PR.

Reply:
Done.